hack/prepare.sh

```diff
@@ -11,13 +11,11 @@ git grep -l 'template' -- ${REPLACE_FILES} | xargs sed -i.bak "s/upjet-provider-
 # shellcheck disable=SC2086
 git grep -l 'template' -- ${REPLACE_FILES} | xargs sed -i.bak "s/template/${PROVIDER_NAME_LOWER}/g"
 # shellcheck disable=SC2086
-git grep -l "upbound/provider-${PROVIDER_NAME_LOWER}" -- ${REPLACE_FILES} | xargs sed -i.bak "s|upbound/provider-${PROVIDER_NAME_LOWER}|${ORGANIZATION_NAME}/provider-${PROVIDER_NAME_LOWER}|g"
-# shellcheck disable=SC2086
```

Comment:
Let's not remove these lines so that files other than Makefile are not affected.

Reply:
Sure @turkenf, I agree.